app/models/ability.rb

```diff
@@ -48,11 +48,13 @@ def caseworker_abilities
     can_search
     can_manage_links
     can_manage_self
+    can_show_defendant
   end
 
   def manager_abilities
     can_search
     can_manage_links
+    can_show_defendant
```

Comment:
[see](https://github.com/ministryofjustice/laa-court-data-ui/pull/231/files#r464490267)

Reply:
👍 Added can %i[show], CourtDataAdaptor::Resource::Defendant to can_search